tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1619,7 +1619,7 @@ class MklQuantizedConv2DOp
                                 this->cpu_engine_);
 
         Tbias *bias_data = reinterpret_cast<Tbias*>(scaled_bias_->get_data_handle());
-        if (!is_bias_const_) bias_data;
+        if (!is_bias_const_) return bias_data;
 
         CacheBias(context, conv_fwd_pd, bias_data, scaled_bias_);
```

Comment:
Please return early to avoid another lookup in `GetCachedBias` below (when bias can be cached), i.e., instead of,
```c++
  if (!is_bias_const_) return bias_data;
  CacheBias(context, conv_fwd_pd, bias_data, scaled_bias_);
}
GetCacheBias(...);
```
Please use
```c++
  if (is_bias_const_) CacheBias(context, conv_fwd_pd, bias_data, scaled_bias_);
  return bias_data;
}
GetCacheBias(...);
```

Reply:
Done.